share/zfsnap/commands/send.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+
+# This file is licensed under the BSD-3-Clause license.
+# See the AUTHORS and LICENSE files for more information.
+
+SSH_KEY=~/.ssh/id_rsa
+#EMAIL=none@none
+user=$USER
+LOCAL='false'
+FULL='false'
+REVERSE='false'
+
+# FUNCTIONS
+Help() {
+    cat << EOF
+${0##*/} v${VERSION}
+
+Syntax:
+${0##*/} destroy [ options ] zpool/filesystem ...
+
+OPTIONS:
+  -b           = source (to be backed up) zfs pool and filesystem, pool_name/filesystem_name
+  -d           = destination (to store backup) zfs pool and filesystem, pool_name/filesystem_name
+  #-e		   = email address to send notification to
+  #-f 		   = enable sendin full backup rather than incremental
```

Comment:
This might be too dangerous for us to use. As far as I can see, the only situations where a full send would be needed would require the use of `-F` on the receiving end, which destroys stuff. In that situation, it's best to leave it to the admin to shoot themselves in the foot rather than us helping.